src/components/Menu/components/MenuAction/MenuAction.scss

```diff
@@ -1,28 +1,24 @@
+@import '../../tokens';
+
 $difference-between-touch-area-and-backdrop: control-height() -
   control-slim-height();
-$action-horizontal-padding: (1.5 * spacing(tight));
 
-.Action {
+.MenuAction {
   @include unstyled-button;
   @include text-style-body;
   position: relative;
   display: inline-flex;
   align-items: center;
+  padding: 0 $menu-action-padding-x;
   min-height: control-height();
-
-  // Negative margin vertically for the amount that is not covered by
-  // the focus overlay
-  margin-top: -0.5 * $difference-between-touch-area-and-backdrop;
-  margin-bottom: -0.5 * $difference-between-touch-area-and-backdrop;
-  padding: 0 $action-horizontal-padding;
```

Comment:
Why does this get removed?

Reply:
At least in my own tophatting, these styles were causing misalignment, not fixing it.